templates/web/pages/bmviii_apply.tmpl

```diff
@@ -0,0 +1,431 @@
+                        </div>
+					</div>
+
+					<h3 style="font-family: Cabin">3. Professional Information</h3>
+					<div class="field">
+                        <label for="github-username" class="label">
+                            <p class="bmviii-app-font" style="margin-bottom: 0px;">
+                                GitHub Username
+                            </p>
+                        </label>
+                        <div class="field-body">
+                            <div class="field">
+                                <div class="control">
+                                    <input type="text" name="github-username" style="background-color: #F0F0F0" class="input" placeholder="smith0987" maxlength="512" value="{{ .FormRefill.Github }}" required>
+                                </div>
+                            </div>
+                        </div>
+					</div>
+
+					<div class="field">
+                        <label for="school" class="label">
+                            <p class="bmviii-app-font" style="margin-bottom: 0px;">
+                                Resume
+                            </p>
+                        </label>
+                        <div class="field-body">
+                            <div class="file has-name">
+                                <label class="file-label">
+                                    {{ if .FormRefill.ResumeFile }}
+                                    <input class="file-input" type="file" accept=".pdf" name="resume">
+                                    {{ else }}
+                                    <input class="file-input" type="file" accept=".pdf" name="resume" required>
+                                    {{ end }}
+                                    <span class="file-cta" style="background-color: #6E9CFF;">
+                                        <span class="file-label" style="font-family: Cabin; font-weight: bold; color: white">
+                                            Upload...
+                                        </span>
+                                    </span>
+                                    {{ if .FormRefill.ResumeFile }}
+                                    <span class="file-name">{{ .FormRefill.ResumeFile }}</span>
+                                    {{ else }}
+                                    <span class="file-name" style="font-family: Cabin; color: #989898; background-color: #F0F0F0">No file selected</span>
+                                    {{ end }}
+                                </label>
+                            </div>
+                        </div>
+					</div>
+
+					<div class="field">
+                        <div class="field-label is-normal">
+                            <label for="is-first-hackathon" class="label">
+                                <p class="bmviii-app-font" style="margin-bottom: 0px">
+                                    Have you attended a hackathon before?
```

Comment:
This dropdown isn't as wide as the others
![Screen Shot 2020-10-20 at 12 31 57](https://user-images.githubusercontent.com/10781907/96616246-f13c8800-12c7-11eb-9336-b14ce490168f.png)